bower.json

```diff
@@ -36,7 +36,7 @@
     "upgradeGuide.md"
   ],
   "dependencies": {
-    "backbone": "1.0.0 - 1.2.3",
+    "backbone": "1.0.0 - 1.3.2",
     "underscore": "1.4.4 - 1.8.3",
     "backbone.babysitter": "^0.1.0",
     "backbone.wreqr": "^1.0.0",
```

Comment:
seems like it should be removed:

https://github.com/marionettejs/backbone.marionette/commit/4c2cc6307a636e9a44ddb8b5385241d8c5429ea1#commitcomment-5873557